stories/ConferenceCard.stories.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { storiesOf, addDecorator } from '@storybook/react'
+import Theme from './theme'
+import ConferenceCard from '../src/components/Events/ConferenceCard'
+import { Grid } from '../src/components/grid'
+
+addDecorator(Theme)
+
+const event = {
+  status: 'Upcoming',
+  date: '21 August 2019',
+  eventName: 'GraphQL helps rockstar developers',
+  eventLocation: 'London',
+  blurb:
+    'GraphQL is an easy way to consume information. It is great. It can do CRUD. It is a perfect tool for rockstar devleopers. This conferencec will feature a number of excellent talks from GraphQL gurus who will show you their tips and tricks to get the best experience possible',
+  homepage: 'www.event.com',
+  ctaLink: 'www.event.com/tickets',
+  svgImage: {
```

Comment:
gatsby image doesn't support svg images so this will just be image instead, can you change it here and where we use it in the ConferenceCard https://github.com/yldio/yld.io/pull/353/files#diff-c12cbaaa077445ec945f2cece1dd3555R83